keyboards/le_chiffre/config.h

```diff
@@ -42,5 +57,24 @@
 #define ENCODERS_PAD_A { D5 }
 #define ENCODERS_PAD_B { D3 }
 
-/* #define RGB_DI_PIN F0
-   #define RGBLED_NUM 1  */
+/*
+ * WS2812 Underglow Matrix options
+ */
+#define RGB_DI_PIN F0
+#define RGBLED_NUM 11
+#define RGBLIGHT_LED_MAP { 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10 }
```

Comment:
I might be missing something, but wouldn't this produce the same as it not being set?
```suggestion
```